cpu/atmega1284p/include/atmega_pcint.h

```diff
@@ -0,0 +1,7 @@
+#ifndef ATMEGA_PCINT_H
+#define ATMEGA_PCINT_H
+#define ATMEGA_PCINT_MAP_PCINT0 { GPIO_PIN(0, 0), GPIO_PIN(0, 1), GPIO_PIN(0, 2), GPIO_PIN(0, 3), GPIO_PIN(0, 4), GPIO_PIN(0, 5), GPIO_PIN(0, 6), GPIO_PIN(0, 7) }
+#define ATMEGA_PCINT_MAP_PCINT1 { GPIO_PIN(1, 0), GPIO_PIN(1, 1), GPIO_PIN(1, 2), GPIO_PIN(1, 3), GPIO_PIN(1, 4), GPIO_PIN(1, 5), GPIO_PIN(1, 6), GPIO_PIN(1, 7) }
+#define ATMEGA_PCINT_MAP_PCINT2 { GPIO_PIN(2, 0), GPIO_PIN(2, 1), GPIO_PIN(2, 2), GPIO_PIN(2, 3), GPIO_PIN(2, 4), GPIO_PIN(2, 5), GPIO_PIN(2, 6), GPIO_PIN(2, 7) }
+#define ATMEGA_PCINT_MAP_PCINT3 { GPIO_PIN(3, 0), GPIO_PIN(3, 1), GPIO_PIN(3, 2), GPIO_PIN(3, 3), GPIO_PIN(3, 4), GPIO_PIN(3, 5), GPIO_PIN(3, 6), GPIO_PIN(3, 7) }
+#endif
```

Comment:
We need the special case anyway, and the current implementation is even better (memory-wise).
Should we move the definitions to the cpu.h anyway?

Reply:
> We need the special case anyway

Yes, and also for non-ATmega2560 while there is a 1:1 mapping for GPIO port and PCINT bank, these mapping are different. I'd personally often prefer less complexity over RAM/ROM improvement.

> the current implementation is even better (memory-wise).

This is a surprise to me. I'd assume in case of the ATmega328p and the ATmega1284p only the port would need to be saved per bank, and the pin can be obtained via `GPIO_PIN(port_of_bank(bank), pin_number)`. But as said, I'd personally would keep it. But feel free to squeeze more ROM bytes out of it, if you see this differently.

> Should we move the definitions to the cpu.h anyway?

I do like the current place. But I do not have a strong opinion about that anyway.